docs/core/development/security.rst

```diff
@@ -1,27 +1,441 @@
+Report a security
+=================
 
-:LastChangedDate: $LastChangedDate$
-:LastChangedRevision: $LastChangedRevision$
-:LastChangedBy: $LastChangedBy$
+We take security very seriously.  Your input and feedback on our security is always appreciated.
 
-Security
-========
+You can report a security issue by creating `a security advisory using the GitHub UI <https://github.com/twisted/twisted/security/advisories/new>`_.
```

Comment:
You almost certainly can't.  You need privileges on the project to file a security advisory against it.

Reply:
True. Thanks. I will update to take this into consideration.